prusti-server/src/client.rs

```diff
@@ -30,6 +30,27 @@ impl PrustiClient {
     pub async fn verify(
         &self,
         request: VerificationRequest,
+    ) -> reqwest::Result<VerificationResult> {
```

Comment:
I suggest creating a version of `VerificationResult` that is not Viper dependent and using it here instead.

Reply:
I moved the `VerificationResult` struct to another crate `backend-common`. The `viper` crate and my (future) native verifier crate depend on it.